src/arch/armv8/vgicv3.c

```diff
@@ -129,7 +129,8 @@ void vgicd_emul_router_access(struct emul_access* acc, struct vgic_reg_handler_i
     struct vgic_int* interrupt = vgic_get_int(cpu()->vcpu, irq_id, cpu()->vcpu->id);
 
     if (interrupt == NULL) {
-        return vgic_emul_razwi(acc, handlers, gicr_access, vgicr_id);
+        vgic_emul_razwi(acc, handlers, gicr_access, vgicr_id); //Check again
```

Comment:
Don't get what the comment is there for.

Reply:
Fixed in commit 334634b63fc85d1f5e110a32f46d95ab8ddea2a5